LittleDatabaseApp/Program.cs

```diff
@@ -0,0 +1,194 @@
+﻿using LittleDatabaseApp.Enums;
+using SubjectsDAL.EF;
+using SubjectsDAL.Models;
+using SubjectsDAL.Repos;
+using System;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LittleDatabaseApp
+{
+    class Program
+    {
+        public static bool exitApp;
+
+        public static List<Human> humans = new List<Human>();
+        public static List<Android> androids = new List<Android>();
+
+        static void Main(string[] args)
+        {
+            Database.SetInitializer(new DataInitializer());
+
+            while (!exitApp)
+            {
+                DisplayMenu();
+            }
+        }
+
+        private static void DisplayMenu()
+        {
+            Console.WriteLine($"Choose option\n1) Add Person\n2) Show peoples in memory\n3) Add people to database\n4) Show database records\n5) Exit App");
+
+            string userInput = Console.ReadLine();
+
+            if (userInput == "5")
+            {
+                exitApp = true;
+                return;
+            }
+
+            RecognizeUserInput(userInput);
+
+            Console.ReadLine();
+            Console.Clear();
+        }
+
+        private static void RecognizeUserInput(string userInput)
+        {
+            MenuOptions MenuChoosen = new MenuOptions();
+
+            if (!int.TryParse(userInput, out int userNumberInput))
+            {
+                    Console.WriteLine("Can't recognize answer");
+            }
+
+            if (userNumberInput == 1 || userNumberInput == 2 || userNumberInput == 3 || userNumberInput == 4)
+            {
+                MenuChoosen = (MenuOptions)userNumberInput - 1;
+                ChooseRightMenu(MenuChoosen);
+            }
+            else
+            {
+                Console.WriteLine("Choose the correct number");
+            }
+        }
+
+        private static void ChooseRightMenu(MenuOptions menu)
+        {
+            switch (menu)
+            {
+                case MenuOptions.addHuman:
+                    Console.Clear();
```

Comment:
Instead of adding at every case `Console.Clear();` use it before the `switch` statement.